ProcessMaker/Models/Process.php

```diff
@@ -1702,8 +1702,10 @@ public function scopeFilter($query, $filterStr)
             $query->where('processes.name', 'like', $filter)
                  ->orWhere('processes.description', 'like', $filter)
                  ->orWhere('processes.status', '=', $filterStr)
-                 ->orWhere('user.firstname', 'like', $filter)
-                 ->orWhere('user.lastname', 'like', $filter)
+                 ->with(['user' => function ($query) use ($filter) {
```

Comment:
hi @marcoAntonioNina this change causes the search in `/processes`by "user" to stop working. I believe this query would be more suitable for the cases we handle:

```
->orWhereHas('user', function ($query) use ($filter) {
    $query->where('firstname', 'like', $filter)
          ->orWhere('lastname', 'like', $filter);
})
```

Reply:
Thanks @eiresendez , the change was updated.